PROCESSES.md

```diff
@@ -2,10 +2,100 @@
 
 This document outlines processes and procedures for some common tasks in the charts repository.
 
+## Review Process
+
+One of the Chart maintainers should review the PR.
+If everything is fine then the PR should be [approved](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/approving-a-pull-request-with-required-reviews).
+The one who approves the PR should also merge it directly.
+In case the reviewer wants someone else to have a look on it it should be mentioned as a comment so that it's transparent for everyone.
+
+As a chart maintainer can not approve it's own PRs every chart should have at least two maintainers.
+For charts where this is not the case or where none of the other maintainers does a review within two weeks the maintainer who created the PR could request a review from a repository admin instead.
```

Comment:
As suggested in https://github.com/prometheus-community/helm-charts/pull/68#issuecomment-695843546 it might be worth to add a timewindow how long repo admin should wait for the original maintainer to approve.

Imho 2 weeks should be maximum.

Reply:
Do you have a suggestion how to make the two weeks more clear than what is already stated?

Reply:
Woops! I've been looking for a number in the text :D 
Sorry.